config/ember-try.js

```diff
@@ -9,6 +9,7 @@ module.exports = function() {
     getChannelURL('canary')
   ]).then((urls) => {
     return {
+      useYarn: true,
```

Comment:
We shouldn't need to migrate to `yarn` just for this change. I don't mind using `yarn` in this repo, but this should be separate into a second PR focused on "migrating to yarn".

Reply:
okay will open that PR.